### DISCUSSION TITLE
Plugin URL

### DISCUSSION THREAD

**Discussion by skinfrakkiirid** [grafana/grafana-infinity-datasource#638] [General]
The current plugin doesn't have a URL configuration:

![image](https://github.com/yesoreyeram/grafana-infinity-datasource/assets/120418829/e82e92c1-ba4a-4add-b311-e666c9962dc0)

**Comment by yesoreyeram** [comment]
It is under misc section now. Mostly you don't need to enter URL there. Instead enter full URL in the query editor.
